lib/core/src/Cardano/Wallet/Primitive/Types.hs

```diff
@@ -705,7 +705,7 @@ fromFlatSlot n = SlotId e (fromIntegral s)
     s = n `mod` epochLength
 
 epochLength :: Integral a => a
-epochLength = 21600
+epochLength = 500
```

Comment:
why this change?

Reply:
This was temporary. Jormungandr uses `500`, so without this the "slotIdTime of tip is recent"-test on jormungandr didn't pass. So we need to at least use a dynamic `SlotsPerEpoch`/`epochLength` from `BlockchainParameters` 🤔